frontend/src/components/documentation/Routes/Timetable/GetPersonalTimetable.jsx

```diff
@@ -102,6 +102,12 @@ export default class GetPersonalTimetable extends React.Component {
                 requirement="required"
                 example="uclapi-5d58c3c4e6bf9c-c2910ad3b6e054-7ef60f44f1c14f-a05147bfd17fdb"
                 description="Authentication token." />
+              <Cell
+                name="Date"
```

Comment:
```suggestion
                name="date"
```
I'd also note this optional field is still before a required field.